### ISSUE TITLE
DirectLine welcome message from standard bot and examples

### ISSUE THREAD

**Issue by vglu** [microsoft/BotFramework-WebChat#3195]
Hi, 

Author ask me to open new issue if I have got something new 
This is continue for 
https://github.com/microsoft/BotFramework-WebChat/issues/3187#event-3394224413

I have got standard QnA maker setup. Publish bot to azure. Add DirectLine channel.

download https://github.com/Microsoft/BotBuilder-Samples.git

go to samples/javascript_es6/70.styling-webchat

setup my secret

```json
      window.WebChat.renderWebChat({
        directLine: window.WebChat.createDirectLine({   secret: 'secret from directline channel' }),
        // directLine: window.WebChat.createDirectLine({ token }),
        styleOptions
      }, document.getElementById('webchat'));
``` 

run `npx serve`

when I open bot page 'localhost:5000' - I have no saw welcome message. 

What I expect to get: Welcome message from QnA bot

Actual result: I have got welcome message ONLY after send first message.

![ice_screenshot_20200603-140021](https://user-images.githubusercontent.com/671915/83629365-a03f2800-a5a2-11ea-9cbe-cbcd08194485.png)
 

I think we should fix example or fix standard bot for QnA maker.
Same behavior with 01.getting-started/g.hybrid-react-npm/ example

Thanks

Regards,
Vitaliy

**Comment by vglu** [comment]
Hi @tdurnford 
Thank you for your answer and link. With BotFramework-WebChat\samples\04.api\a.welcome-event works fine :), 
Thank you for your patience.
